pkg/providers/tinkerbell/reconciler/reconciler.go

```diff
@@ -31,6 +31,18 @@ type IPValidator interface {
 	ValidateControlPlaneIP(ctx context.Context, log logr.Logger, spec *c.Spec) (controller.Result, error)
 }
 
+// Scope object for Tinkerbell reconciler.
+type Scope struct {
```

Comment:
tinkerbell scope need a operation bool at least

Reply:
Why is it called scope?